internal/precache/workload/pullImages.go

```diff
@@ -158,9 +172,18 @@ func PullImages(precacheSpec []string, authFile string) *precache.Progress {
 func ValidatePrecache(status *precache.Progress, bestEffort bool) error {
 	// Check pre-caching execution status
 	if status.Failed != 0 {
+		var imagesFound []string
 		log.Info("Failed to pre-cache the following images:")
 		for _, image := range status.FailedPullList {
-			log.Info(image)
+			if strings.Contains(image, "@sha:") && podmanImgExists(image) {
```

Comment:
nit: We probably don't need the sha check here. That would be more useful if we were determining a failure action when pulling. Since we're just double-checking failed images, we could just do an existence check for any failed.